packages/integration-tests/tests/__snapshots__/v13-new-workspace-create-application-false-ng-add-then-project.test.ts.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`v13-new-workspace-create-application-false-ng-add-then-project it should pass linting when adding a project before running ng-add 2`] = `
 Object {
-  "@angular-devkit/build-angular": "^15.0.2",
+  "@angular-devkit/build-angular": "^15.0.3",
```

Comment:
These snapshot updates were needed to get the CI green 🤷‍♂️

Reply:
Hi @mattlewis92 I have removed this element of flakiness by adding a custom snapshot serializer.

Please could you fix up the conflicts with the latest?

Reply:
Done!